### PR TITLE
addition to chapter "Commonality of Bitcoin and Lightning"

### PR DESCRIPTION
**Pull request by 8go**
- as contrast to chapter of differences between Bitcoin and Lightning
- a lot is missing here
- this is just an initial draft to get the ball rolling and other contributors can add
- even if these things have been mentioned somewhere throughout the book it is useful to gather them here as a summarized list. People looking thru the TOC can find this chapter and it will give them (hopefully) a good overview of commonalities

### CODE REVIEW

File: ch03.asciidoc [hunk 1]
```diff
@@ -556,3 +556,33 @@ When the channel is closed, the final balance of the channel is submitted to the
 For instance: if the opening balance was 1 BTC to Alice, and Alice made a payment of 0.3 BTC to Bob, then the final balance of the channel is 0.7 BTC to Alice and 0.3 BTC to Bob.
 If Alice tries to cheat by submitting the opening state of the channel to the Bitcoin blockchain, with 1 BTC to Alice and 0 BTC to Bob, then Bob can retaliate by submitting the true final state of the channel, as well as penalty transaction that gives him all Bitcoin in the channel.
 In this context, the Bitcoin blockchain acts as a court system; recording the initial and final balances of each channel, and approving penalties if one of the parties tries to cheat.
+
+
+=== Commonality of Bitcoin and Lightning
+
+==== Monetary Unit
+
+Both the Bitcoin network and the Lightning network use the same monetary units: bitcoins.
```
Comment (Roasbeef) [inline ch03.asciidoc:600]: Slight nuance here, but it's possible to pay a "milli satoshi", though this granularity of payment can't be manifested _on-chain_.

File: ch03.asciidoc [hunk 2]
```diff
@@ -556,3 +556,33 @@ When the channel is closed, the final balance of the channel is submitted to the
...
+
+==== Irrevocability 
+
+Both, Bitcoin transactions and Lightning payments are irrevocable. 
+There is no "undo" operation for either one. 
+As a user of either one you have to act responsibly.
+
+=== Trustlessness
+
+Just as Bitcoin, Lightning requires the user only to trust mathematics, encryption and that the Lightning code does not have any critical bugs.
```
Comment (Roasbeef) [inline ch03.asciidoc:611]: Lot of missing nuance here...

As an alternative, we could say something along that lines of "the security model of Lightning reduces to the security of the base blockchain.

**Review by Roasbeef — CHANGES_REQUESTED** [review]
Needs a rebase as well.

File: ch03.asciidoc (hunk 1 above)
Comment (8go) [reply ch03.asciidoc:600]: I added that as a distinction. I had that on the back of my mind already.

See Commit # 3.

File: ch03.asciidoc (hunk 2 above)
Comment (8go) [reply ch03.asciidoc:611]: Fair and correct. Your suggestion is worked in now.

See Commit # 3

**Comment by 8go** [comment]
See Commit # 3 93fea5e37f4930c2c235098666ae23992f6a1122

**Comment by 8go** [comment]
Resolved conflicts in Commit # 4 in 93fea5e37f4930c2c235098666ae23992f6a1122

**Review by renepickhardt — APPROVED** [review]
I really love this section and would have written something similar at some point in time!

**Pull request merged**